app/src/components/account/UserMenu.vue

```diff
@@ -158,7 +186,17 @@ export default {
       )
     },
   },
+  created() {
+    // getAddressFromExtension needs some  time to grab the addresses from extension in the first load
+    setTimeout(() => this.$store.dispatch(`getAddressesFromExtension`), 1000)
+    this.$store.dispatch(`loadAccounts`).then(() => {
```

Comment:
we should rename this to `loadLocalAccounts`

Reply:
Done